INSTALL.md

```
@@ -0,0 +1,54 @@
+## From Source (using Cargo)
+
+```sh
+cargo install dela
+dela init
+```
+
+## Pre-built Binaries
+
+You can download pre-built binaries for your platform from the [GitHub Releases page](https://github.com/yourusername/dela/releases).
+
+### Linux (x86_64)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)